src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+    - `new`: mark the channel read-only
+    - `old`: destroy the channel object
+]])
+
+M['compat.fiber_slice_default'] = format_text([[
+    Define the maximum fiber execution time without a yield:
+
+    - `new`: `{warn = 0.5, err = 1.0}`
+    - `old`: infinity (no warnings or errors raised)
+]])
+
+M['compat.json_escape_forward_slash'] = format_text([[
+    Whether to escape the forward slash symbol "/" using a backslash
+    in a `json.encode()` result:
+
+    - `new`: do not escape the forward slash
+    - `old`: escape the forward slash
+]])
+
+M['compat.replication_synchro_timeout'] = format_text([[
+    The `compat.replication_synchro_timeout` option controls transaction
+    rollback due to `replication.synchro_timeout`.
+
+    - `old`: the old behavior: unconfirmed synchronous transactions are rolled
+    back after a `replication.synchro_timeout`.
+    - `new`: A synchronous transaction can remain in the synchro queue
+    indefinitely until it reaches a quorum of confirmations.
+    `replication.synchro_timeout` is used only to wait confirmation
+    in promote/demote and gc-checkpointing. If some transaction in limbo
+    did not have time to commit within `replication_synchro_timeput`,
+    the corresponding operation: promote/demote or gc-checkpointing
+    can be aborted automatically.
+]])
+
+M['compat.sql_priv'] = format_text([[
+    Whether to enable access checks for SQL requests over iproto:
+
+    - `new`: check the user's access permissions
+    - `old`: allow any user to execute SQL over iproto
+]])
+
+M['compat.sql_seq_scan_default'] = format_text([[
+    Controls the default value of the `sql_seq_scan` session setting:
+
+    - `new`: false
+    - `old`: true
+]])
+
+M['compat.wal_cleanup_delay_deprecation'] = format_text([[
+    Whether option 'wal_cleanup_delay' can be used.
+
+    The old behavior is to log a deprecation warning when it's used, the new
+    one - raise an error.
+
+    If Tarantool participates in a cluster, xlogs needed for other replicas will
+    be retained by persistent WAL GC.
+]])
+
+M['compat.yaml_pretty_multiline'] = format_text([[
+    Whether to encode in block scalar style all multiline strings or ones
+    containing the `\n\n` substring:
+
+    - `new`: all multiline strings
+    - `old`: only strings containing the `\n\n` substring
+]])
+
+-- }}} compat configuration
+
+-- {{{ config configuration
+
+M['config'] = format_text([[
+    The `config` section defines various parameters related to
+    centralized configuration.
+]])
+
+-- {{{ config.context configuration
+
+M['config.context'] = format_text([[
+    Specify how to load settings from external storage. For example,
+    this option can be used to load passwords from safe storage. You can
+    find examples in the Loading secrets from safe storage section.
+]])
+
+M['config.context.*'] = format_text([[
+    The name of an entity that identifies a configuration value to load.
+]])
```

Comment:
Since we describe values now (not keys), I think that this description should be adjusted.

Reply:
BTW, I would call it 'context variable', 'template variable' or just 'variable'. The 'configuration value' term seems a bit misleading for me.

Reply:
> Since we describe values now (not keys), I think that this description should be adjusted.

I think in this case everything is correct and the name should be described here.
Example:
```yaml
config:
  context:
    foo:
      from: env
      env: FOO
```
<img width="439" alt="Снимок экрана 2024-11-19 в 17 52 19" src="https://github.com/user-attachments/assets/ee2ba06e-8165-4a07-8058-960b83e3c8b6">

Reply:
I choose `context`. Maybe like this?
```lua
I['config.context.*'] = format_text([[
    A context variable definition that specifies how
    to load it (e.g. from a file or an environment variable).
]])
```